packages/manager/apps/web-office/src/pages/dashboard/users/ModalChangePasswordUsers.component.tsx

```diff
@@ -36,19 +36,21 @@ export default function ModalChangePasswordUsers() {
   const [searchParams] = useSearchParams();
   const activationEmail = searchParams.get('activationEmail') || '';
   const { addError, addSuccess } = useNotifications();
-  const [shouldSendMail, setShouldSendMail] = useState(true);
+  const [shouldSendMail] = useState(true);
```

Comment:
```suggestion
  const [shouldSendMail] = useState(true);
```
you are sure you need ? because it's relate to email field if not empty and here you don't have set fonction is always true